kotlin/build.gradle.kts

```diff
@@ -1,14 +1,16 @@
 // In this section you declare the dependencies for your production and test code
 dependencies {
+	val versionKotlin = "1.6.0"
```

Comment:
Depends on how dependabot processes it. It would make sense if it just "runs" the gradle file and captures all the dependencies. But that's just an assumption.

Reply:
let me check first if dependabot can see this variable. I will keep it update in this comment.

Reply:
On your edit, if it doesn't support the version file that was mentioned in the MR, I'm probably wrong about the support for this after all

Reply:
I don't ruby programmer, soo it is difficult to find

Reply:
I moved it out for now just to make sure. At a later point I might screw around a bit in a repo myself to see if it gets picked up correctly by dependabot.